src/Spec2-Core/SpNotebookPage.class.st

```diff
@@ -70,17 +69,6 @@ SpNotebookPage class >> title: aString icon: anIcon provider: aBlock [
 		yourself
 ]
 
-{ #category : #'instance creation' }
-SpNotebookPage class >> title: aString icon: anIcon provider: aBlock layoutSpec: aSymbol [
-
-	^ self new 
-		title: aString;
-		icon: anIcon;
-		presenterProvider: aBlock;
-		layoutSpec: aSymbol;
-		yourself
-]
-
```

Comment:
This method is called in SpDemoNotebookPresenter>>objectInspectorPage I don't know why it was removed